bin/predictions.py

```diff
@@ -146,13 +149,11 @@
     surface_bag = surface_bag.map(get_nuclearity)
 
     # choose the number of partitions after to use after making adslab combos
-    npartitions = min(bulk_num * adsorbate_num * 4, 1000)
+    npartitions = min(bulk_num * 10, 1000)
```

Comment:
I think this logic is ok - 1000 is related to the max number of tasks you want on a dask scheduler (<100k or so, and there are a bunch of steps). It's also related to the number of workers (>~5-10 steps/worker will mean things are relatively granular; if someone ran with 1000 workers this might need to get adjusted higher.